automation/check_artifact_size.sh

```diff
@@ -16,11 +16,8 @@ fi
 BUILD_DIR="$1"
 ARTIFACT_ID="$2"
 
-# Even our largest .aar should be less than 30M.
-# Seems like a lot? They include compiled rust code for 4 architectures.
```

Comment:
(To be clear, please re-add this. The part about it being for all four architectures is at least non-obvious)